test/integration/suites-windows/windows-service/01-start-server-service

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+source ./common
+
+docker-up spire-base
+
+create-service spire-server C:\\spire\\bin\\spire-server.exe
+start-service spire-server run -config C:/spire/conf/server/server.conf
```

Comment:
is it possible to be consistent and use `\\` or `/` in all places?

Reply:
Sure! nice suggestion